packages/ui/src/components/Select/SSelectBase.vue

```diff
@@ -107,6 +113,7 @@ provide(SELECT_API_KEY, api)
           name="s-select-dropdown-transition"
           eager
           :wrapper-attrs="{ 'class': 'z-10' }"
+          :inner-wrapper-attrs="{ 'class': sameWidthPopper && 'w-full' }"
```

Comment:
I am not sure if setting `:class="false"` is correct from the Vue side.

This is better, imo:

```suggestion
          :inner-wrapper-attrs="{ 'class': { 'w-full': sameWidthPopper } }"
```